lib/telemetry_metrics.ex

```diff
@@ -93,21 +93,18 @@ defmodule Telemetry.Metrics do
 
     * `:event_name` - the source event name. Can be represented either as a string (e.g.
       `"http.request"`) or a list of atoms (`[:http, :request]`). By default the event name is all but
-      the last segment of the metric name;
+      the last segment of the metric name.
     * `:measurement` - the event measurement used as a source of a metric values. By default it is
       the last segment of the metric name. It can be either an arbitrary term, a key in the event's
       measurements map, or a function accepting the whole measurements map and returning the actual
-      value to be used;
-    * `:metadata` - determines part of the event metadata to be published to supported external
-      sources. It defaults to an empty list. It accepts three possible values:
-      * `:all` - all event metadata is pushed;
-      * list of keys, e.g. `[:table, :kind]` - only these keys from the event metadata are used;
-      * one argument function that takes the event metadata and returns the new metadata;
-    * `:tags` - a subset of metadata keys by which aggregations will be broken down. If `:tags`
+      value to be used.
+      * `:tags` - a subset of metadata keys by which aggregations will be broken down. If `:tags`
       are set but `:metadata` isn't, then `:metadata` is set to the same value as `:tags` for
-      convenience. Defaults to an empty list;
+      convenience. Defaults to an empty list.
```

Comment:
I think that sentence above related to :metadata is not needed anymore.

Reply:
Good catch. 👍